src/components/screens/questionnair-screen.tsx

```diff
@@ -0,0 +1,168 @@
+import DeleteForeverIcon from "@mui/icons-material/DeleteForever";
+import EditIcon from "@mui/icons-material/Edit";
+import { Typography, Button } from "@mui/material";
+import { useNavigate } from "react-router-dom";
+import UserRoleUtils from "src/utils/user-role-utils";
+
+/**
+ * Some mock Data.
+ * TODO: This will be replaced with the actual data from the backend.
+ */
+
+const mockQuestionnaires = [
+  {
+    id: 1,
+    title: "Questionnaire 1",
+    description: "Description for Questionnaire 1",
+    status: <CheckCircleIcon sx={{ color: "green" }} />,
+  },
+  {
+    id: 2,
+    title: "Questionnaire 2",
+    description: "Description for Questionnaire 2",
+    status: <CloseIcon sx={{ color: "red" }} />,
+  },
+  {
+    id: 3,
+    title: "Questionnaire 3",
+    description: "Description for Questionnaire 3",
+    status: <CloseIcon sx={{ color: "red" }} />,
+  },
+  {
+    id: 4,
+    title: "Questionnaire 4",
+    description: "Description for Questionnaire 4",
+    status: <CloseIcon sx={{ color: "red" }} />,
+  },
+];
+
+/**
+ * Questionnaire screen component.
+ */
+
+const QuestionnaireScreen = () => {
+  /**
+   * Navigations for buttons for new questionnaire window.
+   */
+  const navigate = useNavigate();
+
+  const handleClickOpen = () => {
+    navigate("/newQuestionnaire");
+  };
+
+  const handleGoHome = () => {
+    navigate("/");
+  };
+
+  const adminMode = UserRoleUtils.adminMode();
+
+  return (
+    <Card
+      sx={{
+        p: 3,
+        width: "100%",
+        display: "flex",
+        flexDirection: "column",
+        height: "100",
+      }}
+    >
+      <Grid container sx={{ flexGrow: 1 }}>
+        <Grid
+          item
+          xs={12}
+          sx={{
+            display: "flex",
+            flexDirection: { xs: "column", sm: "row" },
+            justifyContent: { xs: "center", sm: "space-between" },
+            alignItems: "center",
+            mb: 2,
+          }}
+        >
+          <Grid item xs={12} sm={6} sx={{ alignItems: "center" }}>
+            <Typography variant="h4" justifyContent={"center"}>
+              Current Questionnaires
+            </Typography>
+          </Grid>
+
+          <Grid
+            item
+            xs={6}
+            sm={3}
+            sx={{
+              display: "flex",
+              justifyContent: { xs: "center", sm: "flex-start" },
+            }}
+          >
+            {adminMode ? (
+              <Button
+                variant="contained"
+                color="primary"
+                onClick={handleClickOpen}
+                size="large"
+              >
+                Build New Questionnaire
+              </Button>
+            ) : null}
+          </Grid>
+
+          <Grid
+            item
+            xs={6}
+            sm={3}
+            sx={{
+              display: "flex",
+              justifyContent: { xs: "center", sm: "flex-end" },
+            }}
+          >
+            <Button
```

Comment:
Maybe instead of red button this could be black to keep the ui consistent. It only returns the user to homepage, not deleting or canceling anything so in my opinion it does not need to be red.

Reply:
Changed back button. Now its same style as in VacationRequestScreen